linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/FunctionEmitter.scala

```diff
@@ -2693,6 +2706,15 @@ private class FunctionEmitter private (
     AnyType
   }
 
+  private def genLinkTimeIf(tree: LinkTimeIf): Type = {
+    markPosition(tree)
```

Comment:
The `markPosition` here is a no-op because we never emit an instruction that will be attached to it.